sdk/samples/iot/pnp/pnp_temperature_controller_component.h

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef PNP_TEMPCONTROLLER_COMPONENT_H
+#define PNP_TEMPCONTROLLER_COMPONENT_H
+
+#include "pnp_mqtt_message.h"
+#include <azure/az_core.h>
+
+/**
+ * @brief Processes a command intended for the temperature controller component.  Parses
+ * request, builds response, and responds via MQTT.
+ *
+ * @param[in] hub_client An
+ * #az_iot_hub_client corresponding to this MQTT connection.
```

Comment:
Split lines throughout the header comments.

Reply:
fixed.